src/Umbraco.Tests/Scoping/ScopeEventDispatcherTests.cs

```diff
@@ -350,6 +374,9 @@ private static void OnDoThingFail(object sender, EventArgs eventArgs)
 
         public static event TypedEventHandler<ScopeEventDispatcherTests, SaveEventArgs<decimal>> DoThing3;
 
+        public static event TypedEventHandler<IContentService, PublishEventArgs<IContent>> Test_UnPublished;
+        public static event TypedEventHandler<IContentService, DeleteEventArgs<IContent>> Test_Deleted;
```

Comment:
Adding this breaks the other unit test since now there are 2x events with the same arg type, i'll fix